whois-update/src/main/java/net/ripe/db/whois/update/domain/UpdateMessages.java

```diff
@@ -220,6 +220,11 @@ public static Message cantRemoveOrgAttribute() {
                 "Please contact \"ncc@ripe.net\" to remove this reference.");
     }
 
+    public static Message cantAddorRemoveRipeNccRemarks() {
+        return new Message(Type.ERROR, "Remarks starting with RIPE NCC can only be added/removed by the RIPE NCC.\n" +
```

Comment:
Change error text to : "The "remarks" attribute can only be added or removed by the RIPE NCC".

Reply:
Remove text to contact ncc@ripe.net.